author/topics/building/reference-lookup.adoc

```diff
@@ -23,6 +23,8 @@ the standard document identifier needs to be encoded in a format recognized by R
 
 ISO deliverables:: `ISO(identifier)`, or any identifier prefixed with `ISO`
 
+ISO/IEC Directives:: `ISO/IEC DIR {n}`, `ISO/IEC DIR {n} + {flavor} SUP`, or `ISO/IEC DIR {flavor} SUP`. `n` - is a number of directive, `flavor` - is `ISO` or `IEC`.
```

Comment:
Can you provide the exact listing of the documents provided in the Static Cache? e.g. ISO/IEC DIR IEC SUP? Should the edition year be specified, or is it automatic? Also mention that this function depends on the Static Cache so the user won't be surprised if it is different from the version online?

Reply:
> Should the edition year be specified, or is it automatic?

For now, there aren't the same documents with the same number and different editions. Do you think we could have what documents in the future?